lib/inquisition/outputter/doc.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'html/template'
+require_relative 'documentation/doc_view_helper'
+
+module Inquisition
+  module Outputter
+    class Doc
+      Outputter.declare(self, :stop)
+
+      def initialize(_output); end
+
+      def stop(issues)
+        generate_doc(issues)
+      end
+
+      def generate_doc(_issues)
```

Comment:
Maybe we can remove this method and move all logic to `stop`, what do you think?